Lib/test/test_fcntl.py

```diff
@@ -138,6 +136,39 @@ def test_flock(self):
         self.assertRaises(ValueError, fcntl.flock, -1, fcntl.LOCK_SH)
         self.assertRaises(TypeError, fcntl.flock, 'spam', fcntl.LOCK_SH)
 
+    def test_lockf(self):
+
+        self.f = open(TESTFN, 'wb+')
+
+        self.assertRaises(TypeError, fcntl.lockf, self.f, "foo")
+        self.assertRaises(TypeError, fcntl.lockf, self.f, fcntl.LOCK_UN, "foo")
+        self.assertRaises(ValueError, fcntl.lockf, self.f, -256)
+        self.assertRaises(ValueError, fcntl.lockf, self.f, 256)
+
+        fcntl.lockf(self.f, fcntl.LOCK_EX | fcntl.LOCK_NB)
+
+        pid = os.fork()
+        if pid == 0:
+            rval = 2
```

Comment:
It would be nice to explain these return values. For example, if I understand the test correctly, the test should never return `2`, so it would be nice to document when the value of `rval` will be `2`.